<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositUI</class>
 <widget class="QWidget" name="DepositUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>760</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>1061</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>예금 입니다. 원하시는 금액을 선택해주세요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>760</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>계좌번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>계좌번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>잔고</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>잔고</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>거래후 금액</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>거래후 금액</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>예금주명</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>예금주명</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>50</y>
     <width>702</width>
     <height>947</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="NumLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <widget class="QPushButton" name="num0Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="num4Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="num8Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="num9Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="num6Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="num3Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="num7Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="num00Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="num5Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="num1Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="num2Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
